tests/queries/0_stateless/03254_timeseries_functions.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE ts_raw_data(timestamp DateTime64(3,'UTC'), value Float64) ENGINE = MergeTree() ORDER BY timestamp;
+
+INSERT INTO ts_raw_data SELECT arrayJoin(*).1::DateTime64(3, 'UTC') AS timestamp, arrayJoin(*).2 AS value
+FROM (
+select [
+(1734955421.374, 0),
+(1734955436.374, 0),
+(1734955451.374, 0),
+(1734955466.374, 0),
+(1734955481.374, 0),
+(1734955496.374, 0),
+(1734955511.374, 1),
+(1734955526.374, 3),
+(1734955541.374, 5),
+(1734955556.374, 5),
+(1734955571.374, 5),
+(1734955586.374, 5),
+(1734955601.374, 8),
+(1734955616.374, 8),
+(1734955631.374, 8),
+(1734955646.374, 8),
+(1734955661.374, 8),
+(1734955676.374, 8)
+]);
```

Comment:
This time series doesn't contain resets. Did you manage to take resets into account when calculating `rate()` or `delta()`?